docs/learn/voting-dapp/voting-dapp.mdx

```diff
@@ -0,0 +1,679 @@
+Blockchain and web3 have changed our understanding of transparency, trust and decentralization. One of the most incredible use-case for this technology is the development of web3 dApps(Decentralized Applications) that offers solutions that are secure, verifiable and tamper-proof solutions to our everyday challenges. Among all the use-cases, decentralized voting stands out. Decentralized technologies remove middlemen and ensure transparent vote counts, this voting dApp has the potential to change governance system while enhancing the democratic process.
```

Comment:
This needs front matter, similar to the content I added in a comment on the other guide, and to match the standard format for the repo